provider-utils/awscloudformation/templates/Amplify_Video_Accelerated_Ott_Hls_Ts_Avc_Aac.json

```diff
@@ -0,0 +1,97 @@
+{
+  "Description": "Amplify Video Encoding Template (Apple HLS @ 1080p30)",
```

Comment:
This needs to change to reflect that this is for accelerated transcoding. (This is what shows up in the Amplify Video prompts)